redirect/routes.php

```diff
@@ -574,6 +590,7 @@ class Route {
         '6. Semester' => [],
         'Special'     => [
             'hunger',
+            'eat-api'
```

Comment:
Now travis is definitely failing because of you :wink: 

Fix the missing comma and we should be ready to go :rocket:

Reply:
Here we go: 43e19f16fe4d638f0a251d4a1bf326127e8ca22b